docusaurus/docs/learn/core-concepts/pki.md

```diff
@@ -1,89 +1,56 @@
-specified in the `edge.enrollment.signingCert`.
+* The `edge.enrollment.signingCert` section defines the edge enrollment CA's certificate and private key. 
+* The `identity` section appears at the root level of the configuration file and, optionally, in any `web[]` listener. Each appearance of `identity` defines a TLS server identity. If only the root-level `identity` secion is present, it is used whenever a TLS server identity is needed by the controller.
 
-#### PKI Configuration
+<!-- the identity.cert property will be used to define the client authentication certificate for controller HA at which time we should update this to stop saying it's always a TLS server certificate -->
 
-The identity section of the Ziti Controller configuration is used by the Ziti Controller when connections are
-established to or from other components of a Ziti Network. There are four sections in the identity block:
-`cert`, `server_cert`, `key`, `ca`.
+The controller provides these TLS servers:
 
-**ca**: A file representing a group of certificates with one or more certificate chains terminating at a trust anchor.
-When a Ziti Network component connects to the Ziti Controller and offers a certificate for validation the incoming
-connection is checked to see if it signed by a trust anchor specified in this file.
+* The control plane API is consumed by routers and presents the certificate defined in `identity.server_cert`.
+* One or more web listeners provide bindings for the controller's REST APIs and other web services. Each web listener presents the certificate defined in its own `web[].identity.server_cert` property, or defaults to the root-level `identity.server_cert`.
+
+:::note
+The private key of the web listener to which the client API is bound is used to sign edge enrollment tokens. During enrollment, the identity or router in possession of a token will verify that it was signed by the same key that is backing the server certificate of the controller's client API. This allows the enrolling identity to confirm the client API URL is correct before requesting authentication certificate(s).
+::: 
 
-**key**: Also referred to as the [private key](https://en.wikipedia.org/wiki/Symmetric-key_algorithm). It is generated
-first and used to produce the certificates specified in the `cert` and `server_cert` fields of the Ziti Controller
-configuration file.
+The trust bundle defined in controller config property `identity.ca` is used by the controller to bundle known CAs' certificates. This bundle is downloaded by clients during enrollment from the client API in `/edge/client/v1/.well-known/est/cacerts` and subsequently used to verify controller and router server certificate chains. These known CAs are never used by the controller to verify client certificates which are always from the controller's own edge enrollment CA or a known, verified external CA.

```

Comment:
`external CAs` are called `3rd Party CAs` in the code, API. The CLI calls them simply `ca` (which is unhelpful).

In the name of having a universal domain language, I suggest using 3rd Party CA here.

Reply:
https://github.com/openziti/ziti-doc/pull/592/commits/a693302bc4cb86c09550fccf1bef2cbe156f234c